### PR TITLE
Add shortcut to expand selection

### PR DESCRIPTION
**Pull request by kmcnaught**
Tested shortcut in Windows, took Mac+Linux versions from https://stackoverflow.com/a/2049926

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -543,6 +543,14 @@
                 "key": "f4",
                 "command": "java.action.showTypeHierarchy",
                 "when": "editorLangId == java && editorTextFocus"
+            },
+            {
+                "mac": "shift+cmd+up",
+                "win": "alt+shift+up",
+                "linux": "shift+ctrl+up",
```
Comment (p-) [inline package.json]: The keybinding on Linux seems to be alt+shift+up as well.

File: package.json [hunk 2]
```diff
@@ -543,6 +543,14 @@
...
+                "key": "alt+shift+up",
+                "command": "editor.action.smartSelect.expand",
+                "when": "editorLangId == java && editorTextFocus"
```
Comment (p-) [inline package.json]: Remove editorLangId == java since this keybinding makes sense for other languages as well.

File: package.json [hunk 3]
```diff
@@ -543,6 +543,14 @@
                 "key": "f4",
                 "command": "java.action.showTypeHierarchy",
                 "when": "editorLangId == java && editorTextFocus"
+            },
+            {
+                "mac": "shift+cmd+up",
```
Comment (p-) [inline package.json]: shift+cmd+up is not the right keybinding for macOS. But no keybinding seem to work on Eclipse for macOS. So I would leave the binding to alt+shift+up here as well.

**Review by p- — COMMENTED** [review]
Hi @kmcnaught 
Thanks for your Pull Request!
I added some review comments.

**Comment by kmcnaught** [comment]
@p- thanks for the review, I have made the suggested changes

**Comment by p-** [comment]
@kmcnaught Thanks for making the changes so fast!

**Comment by p-** [comment]
@kmcnaught 
Thanks for you contribution, it was just releasesd as part of Version 0.16.0.

**Pull request merged**